### ISSUE TITLE
Question about pretrain models on 800 epochs

### ISSUE THREAD

**Issue by Lrbomchz** [Haochen-Wang409/HPM#7]
Hello, nice to read this outstanding paper, I have some questions about the settings when pretrain model on 800 epochs, did your settings keep the same as  pretrain model on 200 epoch in **_Table S1 and S2_**?(such as: **pretrain weight decay** 0.05, **pretrain layer-wise lr decay** 0.8, **finetune training epochs** 100)

**Comment by Haochen-Wang409** [comment]
Thanks for your attention!
Yes, the settings are exactly the same.

**Comment by Lrbomchz** [comment]
THANKS!
